src/Route.ts

```diff
@@ -7,7 +7,7 @@ import * as E from './Encoder'
 
 export type GenericRec = Record<string, unknown>
 
-export type EmptyRec = Record<string, never>
+export type EmptyRec = Record<never, never>
```

Comment:
If this is `never` then `EmptyRec & A` === `A` so we can do type equality tests for routes.